helix-core/src/test/java/org/apache/helix/model/cloud/TestCloudConfig.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.helix.model.cloud;
```

Comment:
You may want to check TestHelixConfigAccessor and TestConfigAccessor. You can merge your test into the existing tests, as cloud config is comparable to cluster config, participant config etc, and the test environment is already set up there.

Reply:
I would say let's keep them separated because this config is newly introduced and it is easier to maintain it later. Perhaps we will add more test to this Tests later to this file.